src/main/resources/META-INF/resources/primefaces/jquery/jquery.ui.timepicker.js

```diff
@@ -2250,10 +2259,12 @@
 	 * @return {void}
 	 */
 	$.timepicker.log = function () {
+	    /* PrimeFaces Customization
```

Comment:
This comment block might not be needed since it is part of https://github.com/primefaces/primefaces/commit/5ade2f744b9505ae1c20373c87b88657a9b57c58. I guess what it is trying to do is to not to log errors to console  Since there is no error anyway, we might bring it back.

Reply:
Reverted.  Agreed the console logging should be harmless and should actually show if there is an error